perfkitbenchmarker/vm_util.py

```diff
@@ -77,6 +77,9 @@
 flags.DEFINE_integer('background_cpu_threads', None,
                      'Number of threads of background cpu usage while '
                      'running a benchmark')
+flags.DEFINE_integer('background_network_mbits', None,
+                     'Amount of background network traffic while '
```

Comment:
How about "Rate of background network traffic..."? It would be nice if the per-second component of the unit could also make its way into the flag name, but I can't think of an elegant way to do that since "mbps" could be megabytes per second.